scenarios/csharp/GitHubLovesSparkDotNet/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+using System.Text.RegularExpressions;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Spark.Scenarios
+{
+    public class Program
+    {
+        #region Cloud Run
+        public const string StorageConfigKey
+            = "fs.azure.account.key.<your-storage-account-name>.dfs.core.windows.net";
+        public const string SecureStorageKey = "<your-storage-account-key>";
+        public const string CloudStoragePath
+            = "abfss://<filesystem>@<storage-account>.dfs.core.windows.net/<path>/";
+        #endregion
+
+        #region Local Run
+        public static string LocalDataStoragePath { get; } =
+            Path.Combine(
+                Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
+                "Resources" + Path.DirectorySeparatorChar); 
+        #endregion
+
+        static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                PrintUsage();
+                return;
+            }
+
+            string StoragePath = (args[0] == "local") ? LocalDataStoragePath : CloudStoragePath;
```

Comment:
I like that you're giving users the option for either cloud or local since it lets them use whichever one is easiest/most relevant for their needs.

Reply:
Thanks!